### ISSUE TITLE
Dark Mode

### ISSUE THREAD

**Issue by fennifith** [fennifith/Status#153]
It seems that black text on a while background is in fact bad for accessibility. I was not aware of this, but it should be fixed. And I should pay more attention to this kind of stuff in the future.
